source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -138,6 +140,19 @@ Settings
        | **Default**: ``false``
        | **Accepted Values**: ``true`` or ``false``
 
+   * - | **change.stream.document.key.as.key**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | Whether to use the document key for the source record key if
+         the document key is present. When this setting is ``true``,
+         tombstone events have the keys of the deleted documents.
```

Comment:
Suggestion:
I think this should communicate what is setting them to establish that the setting caused that effect. E.g.

"When set to true, the connector adds keys of the deleted documents to the tombstone events."
"When set to false, the connector adds the resume token as the source key of the tombstone events."